scripts/src/modules/search/search-sort-by.js

```diff
@@ -0,0 +1,9 @@
+export default function () {
+    const sortByForm = document.querySelector("[data-sort-search]");
+    const sortBySelector = sortByForm.querySelector("#id_sort_by_desktop");
+
+    sortBySelector.addEventListener("change", function () {
+        console.log("clicked select");
```

Comment:
**Question:** I'm imagining this console log is not needed.

Reply:
You're correct - thanks!